conf/software-vendors.json

```diff
@@ -182,6 +182,24 @@
       "incomeAndDeductions": [
         "self-employment"
       ]
+    },
+    {
+      "name": "IRIS Accountancy Suite",
+      "url": "https://www.iris.co.uk/products/iris-accountancy-suite/",
+      "phone": "03448155555",
+      "email": "https://www.iris.co.uk/",
```

Comment:
This isn't an email address, can you confirm this with BA?

Reply:
Email address updated